sql/core/src/test/resources/sql-tests/inputs/postgreSQL/int2.sql

```diff
@@ -92,9 +92,11 @@ WHERE f1 > -32767;
 
 SELECT '' AS five, i.f1, i.f1 - int('2') AS x FROM INT2_TBL i;
 
-SELECT '' AS five, i.f1, i.f1 / smallint('2') AS x FROM INT2_TBL i;
+-- PostgreSQL `/` is the same with Spark `div` since SPARK-2659.
+SELECT '' AS five, i.f1, i.f1 div smallint('2') AS x FROM INT2_TBL i;
```

Comment:
let's keep the tests the same as pgsql. we should still use `/`.

Reply:
Copy that, also changed the test result in 17a3c77.